server/game/cards/03_TWI/units/PadawanStarFighter.ts

```diff
@@ -0,0 +1,22 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { Trait } from '../../../core/Constants';
+
+export default class PadawanStarFighter extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '4718895864',
+            internalName: 'padawan-starfighter'
+        };
+    }
+
+    protected override setupCardAbilities() {
+        this.addConstantAbility({
+            title: 'While you control a Force unit or a Force upgrade, this unit gets +1/+1',
+            condition: (context) => context.source.controller.getArenaCards({ trait: Trait.Force }).length > 0,
```

Comment:
Apologies, I missed something - instead of `getArenaCards().length > 0` we should do `hasSomeArenaCard()`